### PR TITLE
HW 44 Polina

### PR DESCRIPTION
**Pull request by polishukova**
I need help with delete from favorites

### CODE REVIEW

File: my-app/src/Store/post.ts [hunk 1]
```diff
@@ -10,9 +11,43 @@ export const postsSlice = createSlice({
...
+        increaseDislike: (state, action: PayloadAction<number>) => {
+            const post = state.posts.find(post => post.id === action.payload)
+            if (!post) return
+            post.dislikes = post.dislikes ? post.dislikes + 1 : 1
+        },
+        setFavoritesPosts: (state, action: PayloadAction<number>) => {
+            const post = state.posts.find(post => post.id === action.payload)
+            if (!post) return
+            state.favorites = [...state.favorites, post]
+            state.favorites.filter(el => el.id !== action.payload)
```
Comment (jathpr) [inline my-app/src/Store/post.ts:28]: why do you need this?

**Comment by polishukova** [comment]
It's works, but so strange way..

File: my-app/src/Components/Tabs/FavoritePosts.tsx [hunk 2]
```diff
@@ -2,7 +2,7 @@ import { useAppSelector } from "../../Store/store"
 import { RenderPost } from "../Post/RenderPost"
 
 export const FavoritePosts = () => {
-    const posts = useAppSelector(state => state.posts.favorites)
+    const posts = useAppSelector(state => state.posts.posts.filter(post => post.favorite===true))
```
Comment (jathpr) [inline my-app/src/Components/Tabs/FavoritePosts.tsx:5]: same as
`const posts = useAppSelector(state => state.posts.posts.filter(post => post.favorite))`

**Comment by jathpr** [comment]
> It's works, but so strange way..

why strange? lgfm

**Pull request merged**